libs/platform/experience/src/mixins/layout.mixin.ts

```diff
@@ -63,19 +69,22 @@ export declare class LayoutMixinInterface {
   lg?: LayoutProperties;
   // @deprecated since 1.2 will be removed, use layoutXl instead
   xl?: LayoutProperties;
+  // @deprecated since 1.2 will be removed from class declaration
+  protected layoutStyles: ConnectableSignal<string | undefined>;
 
   layout?: CompositionLayout | LayoutTypes;
   layoutXs?: LayoutProperties;
   layoutSm?: LayoutProperties;
   layoutMd?: LayoutProperties;
   layoutLg?: LayoutProperties;
   layoutXl?: LayoutProperties;
-  protected layoutStyles: ConnectableSignal<string | undefined>;
   protected renderLayout: (props: LayoutMixinRender) => TemplateResult;
-  protected getLayoutRender(
-    place: keyof LayoutPluginRender,
-    data: LayoutControllerRender
-  ): TemplateResult;
+  protected $preLayoutRenderComposition: ConnectableSignal<
+    Record<string, TemplateResult> | undefined
+  >;
+  protected $postLayoutRenderComposition: ConnectableSignal<
+    Record<string, TemplateResult> | undefined
+  >;
```

Comment:
Are these only used by compositions? Or also intended to be used inside other components, such as product list?

Reply:
i'm not stoked by the names eitherway, any alternatives we could consider?

Reply:
it's only for composition (we need it only for composition-component)

Reply:
it is for composition per se, or for cases with multiple elements in general? 
if it's only for composition, maybe we could extract it to another (sth like layout compostion) mixin, to separate concerns?

Reply:
I was also thinking on additional mixin in this case